pyproject.toml

```diff
@@ -34,29 +34,22 @@ dependencies = [
 [tool.hatch.version]
 path = "src/kagglehub/__init__.py"
 
-[tool.hatch.envs.default]
+[tool.hatch.envs.hatch-test]
 dependencies = [
   "coverage[toml]>=6.5",
   "pytest",
   "flask",
   "flask-jwt-extended",
 ]
 
-[tool.hatch.envs.default.scripts]
-test = "pytest {args:tests}"
-integration-test = "pytest {args:integration_tests}"
-test-cov = "coverage run -m pytest {args:tests integration_tests}"
-cov-report = [
-  "- coverage combine",
-  "coverage report",
-]
-cov = [
-  "test-cov",
-  "cov-report",
-]
+[[tool.hatch.envs.hatch-test.matrix]]
+python = ["3.9", "3.10", "3.11"]
 
 [[tool.hatch.envs.all.matrix]]
-python = ["3.7", "3.8", "3.9", "3.10", "3.11"]
+python = ["3.9", "3.10", "3.11"]
```

Comment:
We should also add 3.12 to match our GCB triggers: https://github.com/Kaggle/kaggleazure/blob/aa1de82d6f1dc71c5715c40b6327053ead4648b7/configs/cicd/cloud_builds/main.tf#L53-L58

Reply:
Done.